src/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -175,7 +172,11 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
             
             if (triggerAction == TDTM_Runnable.Action.AfterInsert)
                 copyRecurringDonationCampaignAllocations(listOppsForProcessing);
-            
+
```

Comment:
@judisohn Came across this particular scenario while testing. I believe this is a bug that we should create a new issue for. What do you think?